dotCMS/src/main/java/com/dotcms/rest/MapToContentletPopulator.java

```diff
@@ -239,6 +254,43 @@ private void fillFields(final Contentlet contentlet,
         }
     } // fillFields.
 
+    private static void processPlainValueForBinaryField(final Map<String, Object> map,
+                                                        final Field field,
+                                                        final Object value,
```

Comment:
This should take a Map<String,String> and not an Object.

Reply:
It is actually a String, Object b/c you can have inner objects inside the json sent by Rest